curriculum/challenges/english/25-front-end-development/workshop-pin-extractor/68592b42b9c7f27cbe123203.md

```diff
@@ -0,0 +1,81 @@
+---
+id: 68592b42b9c7f27cbe123203
+title: Step 15
+challengeType: 20
+dashedName: step-15
+---
+
+# --description--
+
+The third line of the poem is missing a third word, so the pin is shorter than expected.
+
+Add an `else` clause that adds a `'0'` to `secret_code` when there are not enough words, so that all lines of the poem are used to find digits.
```

Comment:
```suggestion
Add an `else` clause that appends a `'0'` to `secret_code` when there are not enough words, so that all lines of the poem are used to find digits.
```